<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ma_emiten">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Emiten</string>
   </property>
  </widget>
  <widget class="QLabel" name="ma_value">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Menggunakan MA</string>
   </property>
  </widget>
  <widget class="QLabel" name="ma_total_transactions">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Transactions</string>
   </property>
  </widget>
  <widget class="QLabel" name="ma_win">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Win:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ma_loss">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Loss:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ma_winrate">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Win Rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ma_detail_transactions">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Detail Transactions:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ma_table_detail">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>631</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Tanggal Beli</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tanggal Jual</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga Beli</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga Jual</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>%Return</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="ma_emiten_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Urutkan:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ma_combobox_urutkan">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Terlama</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Terbaru</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="ma_average_return">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Average Return</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ma_input">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="ma_checkbox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ma_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
